sys/net/gnrc/network_layer/ipv6/nib/nib_abr.c

```diff
@@ -89,12 +89,15 @@ bool gnrc_ipv6_nib_abr_iter(void **state, gnrc_ipv6_nib_abr_t *entry)
 void gnrc_ipv6_nib_abr_print(gnrc_ipv6_nib_abr_t *abr)
 {
     char addr_str[IPV6_ADDR_MAX_STR_LEN];
-    uint32_t now = evtimer_now_min();
+    uint32_t now = evtimer_now_msec();
+
+    /* Some thing seems strage here  does it expire if valid_until in 0 or not*/
+
```

Comment:
@miri64 : is this what should be written here ?

```suggestion
    uint32_t now = evtimer_now_msec();

    uint32_t abr_valid_until = abr->valid_until_ms;

    if(abr_valid_until == 0){
        printf("%s v%" PRIu32 " not expires %" PRIu32 "min\n",
               ipv6_addr_to_str(addr_str, &abr->addr, sizeof(addr_str)),
               abr->version,
               SIXLOWPAN_ND_OPT_ABR_LTIME_DEFAULT);
    } else {
        uint32_t abr_expires_min = (abr_valid_until - now) / (SEC_PER_MIN * MS_PER_SEC);
        printf("%s v%" PRIu32 " expires %" PRIu32 "min\n",
           ipv6_addr_to_str(addr_str, &abr->addr, sizeof(addr_str)),
           abr->version,
           abr_expires_min);
    }

```